src/promptflow/promptflow/batch/_chat_group_orchestrator_proxy.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+from typing import Optional, List, Dict, Mapping, Any
+from pathlib import Path
+from promptflow.contracts.flow import ChatGroupRole
+from promptflow._constants import LANGUAGE_KEY, FlowLanguage
+from promptflow._utils.yaml_utils import load_yaml
+from promptflow.batch._base_executor_proxy import AbstractExecutorProxy
+from promptflow.executor._result import LineResult
+from promptflow.orchestrator._chat_group_orchestrator import ChatGroupOrchestrator
+from promptflow.storage._run_storage import AbstractRunStorage
+
+class ChatGroupOrchestratorProxy(AbstractExecutorProxy):
+    def __init__(
+        self,
+        **kwargs
+    ):
+        self._orchestrator = None
+        super().__init__(**kwargs)
+
+    async def create(
+        cls,
+        flow_file: Path,
+        working_dir: Optional[Path] = None,
+        *,
+        connections: Optional[dict] = None,
+        storage: Optional[AbstractRunStorage] = None,
+        **kwargs,
+    ) -> "AbstractExecutorProxy":
+        """Create a new executor"""
+
+        chat_group_roles = kwargs.get("chat_group_roles", None)
+        max_turn = kwargs.get("max_turn", None)
```

Comment:
You can put them to function param directly, maybe like:
``` python
async def create(
        cls,
        flow_file: Path,
        working_dir: Optional[Path] = None,
        *,
        connections: Optional[dict] = None,
        storage: Optional[AbstractRunStorage] = None,
        chat_group_roles: ...=None,
        max_turn: ...=None,
        run_id: ...=None
)
```

Reply:
Updated